### ISSUE TITLE
This repo is outdated + I have problems understanding the data

### ISSUE THREAD

**Issue by a-berg** [cognitedata/open-industrial-data#10]
Hello everyone, I have recently started to explore this dataset, and have spent a few days on it.
So far, I'd say I'm none the wiser than when I started.

I'd like to know how to discover relevant time-series for e.g. the compressor and tell which ones are inputs and which ones are outputs.
So far, I can't tell neither from your notebooks (I mean... `in_ts_names = ["VAL_23-FT-92512:X.Value", "VAL_23-PT-92512:X.Value", "VAL_23-TT-92502:X.Value"]` and `out_ts_name = "VAL_23-PT-92504:X.Value"` is not very informative) nor from P&ID in Data Fusion.

Moreover, timeseries associated to Asset "23-KA-9101" are things like: "VAL_23-TE-96137-02:X.Value". I go to Dat Fusion and search for it; get to a diagram, find it (manually as there is no highlight), finally I see something about hot air... and I still don't know what this Asset/TimeSeries is for.

TL;DR How do I understand which time series are relevant?

**Comment by rek** [comment]
Perhaps https://hub.cognite.com/open-industrial-data-211 can be helpful.
